src/plugins/scriptedwizard/resources/wxwidgets/wizard.script

```diff
@@ -1100,9 +1095,17 @@ function OnGetNextPage_WxConf()
 function OnEnter_WxConfAdvOpt(fwd)
 {
     if (fwd)
-    {
+    {        
+        Wizard.CheckCheckbox(_T("chkWxDebug"), 
+            IntToBool(GetConfigManager().Read(_T("/wx_project_wizard/wxdebug"), BoolToInt(ChkWxDebug))));
```

Comment:
I don't like all those repeated calls to `GetConfigManager()`...

Reply:
I just followed what was used elsewhere in the wizard, trying to stick to the convention and make minimum changes.

Reply:
No, they are not problematic (in C++ I'd say they are slow, but this is squirrel), but I don't like them because the code becomes really long and really repeatable.

Reply:
I understand but I still do not know if using just `ConfigManager` instead of `GetConfigManager()` is correct, i.e., is there an "instance" of `ConfigManager`?. I checked other scripts, the few I found using it use both. I also did not find the answer in [the docs ](https://wiki.codeblocks.org/index.php/Scripting_commands#Application_object).

If there is `ConfigManager`, I will use it, otherwise I will do something like this `local config_manager = GetConfigManager();` in cases where there would be multiple calls to the method.

However, please note that such calls are very common in the existing wizard code and not only with `GetConfigManager()` but for example also with `GetCompilerFactory()`.

Reply:
1. The idea is to call GetConfigManager to get an instance and use it instead. The `ConfigManager.Something(...)` works by accident I think. I'm not the original author of the binding, but I think it is not correct.

2. Please don't use underscores in names, use camelCase. 

3.
> However, please note that such calls are very common in the existing wizard code and not only with `GetConfigManager()` but for example also with `GetCompilerFactory()`.

I know and I think it is a bad practice, which should not be continued... No need to fix all places which do this, just don't introduce new uses of this pattern/idiom.